sources/packages/api/src/esdc-integration/models/esdc-integration.model.ts

```diff
@@ -0,0 +1,24 @@
+export const DATE_FORMAT = "YYYYMMDD";
+export const SPACE_FILLER = " ";
+export const NUMBER_FILLER = "0";
+export const TIME_FORMAT = "HHmm";
```

Comment:
Name can be more specific to the format of the date.

Reply:
As we are defining the constants in the specific folder, I believe this can be same.

Reply:
But what if esdc integration involves one more time format in future ?